GSA/digital-gov-drupal
# DIGITAL-37: Migrate links paragraph templates.

## Jira ticket




[DIGITAL-37](https://cm-jira.usa.gov/browse/DIGITAL-37)

## Purpose

Adds paragraph_view_mode module.
Migrate template for featured links field display. Adds separate templates for default and featured links views of resources. Updates featured links template with correct markup. Switch shortcodes to use theme templates for resources.

It can be helpful to understand:
1. What the solution is,
2. Why this approach was chosen,
3. How you implemented the change
4. Possible limitations of this approach and alternate solution paths. -->

## Includes the following PRs that must be merged first



## Deployment and testing

### Local Setup


`bin/drush cim` to get paragraph type updates
`bin/drush cr` to refresh theme template cache

### QA/Testing instructions


Edit a topic page, add one or more internal and external resources
Confirm they use the expected HTML from the featured-links template
Override the title, summary for resources and confirm overrides are used

## Checklist for the Developer




## Checklist for the Peer Reviewers
